src/styles/globals.css

```
@@ -2,151 +2,114 @@
 
 @keyframes typing {
-  from { width: 0 }
-  to { width: 10ch }
+  from { width: 0; }
+  to { width: 10ch; }
 }
 
 @keyframes blink-caret {
-  from, to { border-color: transparent }
-  50% { border-color: #22c55e; }
+  from, to { border-color: transparent; }
```

Stylelint (17.12.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)